staging/src/k8s.io/client-go/testing/fixture_test.go

```diff
@@ -429,33 +430,33 @@ func TestGetWithExactMatch(t *testing.T) {
 	nodeResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "node"}
 	node, gvr := constructObject(nodeResource, "node", "")
 
-	assert.Nil(t, o.Add(node))
+	require.NoError(t, o.Add(node))
```

Comment:
Converting `assert.Nil` to `assert.NoError` is obvious.  Converting it to `require` means I have to read the rest of the test.  Better to do that as a separate PR